flake.nix

```diff
@@ -0,0 +1,17 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs";
+  };
+
+  outputs = {nixpkgs, ...}:
+    let pkgs = nixpkgs.legacyPackages.x86_64-linux;
+    in {
+    devShells.x86_64-linux.default = pkgs.mkShell {
+      buildInputs = with pkgs; [ cabal-install ghc haskell-language-server sundials
```

Comment:
For my curiosity, where is the sundials version specified? I can't see it either in flake.lock

Reply:
it's the version that is available in the nixpkgs locked in flake.lock

Reply:
What about `./CustomSundials/default.nix` then?

Reply:
`CustomSundials/default.nix` is used in `default.nix` which works, but is highly outdated.

In our private codebase, we use `CustomSundials/default.nix`.

Actually, this change (i.e. introducing the `flake.nix`) here is completely irrelevant and could be removed if you wish. However maybe in the future it will evolve and replace `default.nix` and `shell.nix` with something more robust.